tools/proto_format/proto_format.sh

```diff
@@ -6,6 +6,10 @@ set -e
 
 [[ "$1" == "check" || "$1" == "fix" ]] || (echo "Usage: $0 <check|fix>"; exit 1)
 
+./tools/git/modified_since_last_github_commit.sh ./api/envoy proto || \
```

Comment:
This will miss two thing:
1. protoxform/proto* tool changes
2. bzl changes, (i.e. protobuf dependencies).

Reply:
Yeah, this is the reason I added the `FORCE_PROTO_FORMAT` override for folks working with `protoxform`. It seems hard to fully capture the potential set of dependencies in a maintainable way here, so might as well skip and then override for the relatively rare case of tool development work by a small subset of folks. I will add some comments.